internal/factory/container/container.go

```diff
@@ -759,3 +767,98 @@ func getOCICapabilitiesList() []string {
 	}
 	return caps
 }
+
+func (c *container) SpecSetPrivileges(ctx context.Context, securityContext *types.LinuxContainerSecurityContext, cfg *config.Config) error {
```

Comment:
part of the hope of this package is to also add unit tests to test the spec as we modify it. I know this has been sitting for a long time, are you able to add some unit tests here @littlejawa ? if not, we can maybe find another contributor to add them

Reply:
Sure, makes sense.
I'll need a little more time, but if you're ok with waiting a bit, I will look into it